### PR TITLE
#1011 Ad post preview page

### PR DESCRIPTION
**Pull request by Diatrex**

### CODE REVIEW

File: addons/default/visiosoft/advs-module/resources/views/new-ad/preview/preview.twig [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{% extends "theme::layouts/default" %}
+
+{% block content %}
+    <div class="pt-5">
+        <p class="preview-paragraph">
+            {{ trans('visiosoft.module.advs::field.preview_text.part_1') }}
```
Comment (fatihalp) [inline addons/default/visiosoft/advs-module/resources/views/new-ad/preview/preview.twig]: part_1 is meaingless what is it? @Diatrex  Give a name

Comment (Diatrex) [reply addons/default/visiosoft/advs-module/resources/views/new-ad/preview/preview.twig]: the name is preview_text, it has two sentences, I want to put a break line tag between them, there is no meaningful name for them, I can name them sentence 1 and sentences 2

Comment (fatihalp) [reply addons/default/visiosoft/advs-module/resources/views/new-ad/preview/preview.twig]: ok then

File: addons/default/visiosoft/profile-module/src/Http/Controller/CacheController.php [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 {
-    public function getUserInfo(Request $request, UserRepositoryInterface $userRepository)
+    public function getUserInfo()
     {
-        $user = $userRepository->find($request->userId);
+        $user = auth()->user();
+        $response = $user ? $user->first_name . ' ' . $user->last_name : $user;
 
-        return ['userName' => $user->first_name . ' ' . $user->last_name];
+        return ['userName' => $response];
```
Comment (fatihalp) [inline addons/default/visiosoft/profile-module/src/Http/Controller/CacheController.php]: actually it's needed to return userid, and if user has store @Diatrex  Then you can render menu

Comment (fatihalp) [reply addons/default/visiosoft/profile-module/src/Http/Controller/CacheController.php]: actually we don't have cache controller. It's just service that returning user data back @Diatrex

Comment (Diatrex) [reply addons/default/visiosoft/profile-module/src/Http/Controller/CacheController.php]: I removed the addBlock function from the twig file. now we are getting the addblock from the controller

**Pull request merged**